src/types/mod.rs

```diff
@@ -502,6 +502,115 @@ mod rewards_overview {
     }
 }
 
+pub mod block_certificates {
```

Comment:
Please put this in a separate file if you want a module.

We don't generally have public modules inlined like this in a single file. This file is already big enough.